kubeadm/flannel/Dockerfile

```diff
@@ -1,30 +1,43 @@
-ARG servercoreTag="ltsc2019"
-ARG cniVersion="0.8.5"
-ARG golangTag=windowsservercore-1809
+ARG BASE="mcr.microsoft.com/powershell:nanoserver-1809"
+ARG cniVersion="0.8.7"
+ARG flannelVersion="0.13.0"
+ARG winsVersion="0.0.4"
+ARG jqVersion="3.4.1"
 
-FROM golang:${golangTag} as builder
-ADD setup.go build/
-RUN go build -o build/setup.exe build/setup.go
-
-FROM mcr.microsoft.com/windows/servercore:${servercoreTag}
-SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+FROM --platform=linux/amd64 golang as setup
+ENV GOOS=windows
+ENV GOARCH=amd64
+WORKDIR /build
+ADD setup.go .
+RUN go build -o setup.exe setup.go
 
+FROM --platform=linux/amd64 curlimages/curl as bins
 ARG cniVersion
+ARG flannelVersion
+ARG winsVersion
+ARG jqVersion
+
+WORKDIR /utils
+RUN curl -Lo wins.exe https://github.com/rancher/wins/releases/download/v${winsVersion}/wins.exe
+RUN curl -Lo yq.exe https://github.com/mikefarah/yq/releases/download/${jqVersion}/yq_windows_amd64.exe
```

Comment:
Shouldn't the command line switches be `-LO`?

Reply:
No. `-Lo` is correct. `-LO` is a little bit different.

Reply:
OH! You're right. I got `o` and `O` mixed up, thinking `O` was the just download the file and save as the same name as the leaf of the url--but that is `o`.